Purpose: Make sure startpar do not segfault when asked to run scripts
	 in parallel without using the -a argument option.
Fixes:   none
Authour: Petter Reinholdtsen
Status:  Sent upstream.

--- a/startpar.c	(revision 1898)
+++ b/startpar.c	(working copy)
@@ -366,7 +366,7 @@
 
   closeall();
 
-  if (!strcmp(arg, "start")) 
+  if (arg && !strcmp(arg, "start")) 
     { 
       int s, t, len;
       pid_t child;
